packages/utils/src/DOMUtils.ts

```diff
@@ -28,4 +28,29 @@ export function identityExtractHTMLElement<T>(
   return maybeHTMLElement instanceof HTMLElement ? maybeHTMLElement : null;
 }
 
-export default { getClosestByClassName, identityExtractHTMLElement };
+/**
+ * Synchronize CSS animations with a given name to the same start time.
+ * @param animationName
+ * @param startTime
+ */
+export function syncAnimationStartTime(
+  animationName: string,
+  startTime: number
+): void {
+  const animations = document
+    .getAnimations()
+    .filter(
+      a => a instanceof CSSAnimation && a.animationName === animationName
+    );
+
+  animations.forEach(a => {
+    // eslint-disable-next-line no-param-reassign
+    a.startTime = startTime;
+  });
+}
```

Comment:
Not sure if I'm reading this incorrectly, but looking at the example on MDN: https://developer.mozilla.org/en-US/docs/Web/API/Animation/startTime#examples
They're setting the new animation to be the same as the existing start time; here you're resetting all of the animations to the same `startTime` (which is always `0`, since that's the only value you pass in). Wouldn't that mean that an existing animation would jump back when a new one is added? Whereas we should just be syncing the new animation to any existing loading spinner that's already shown.

Reply:
My observation when I dug into this is that the `startTime` is usually `null` by default (also was not what I expected from the docs). My understanding is that it's kind of a static value that determines the basis of when animations are calculated. Setting explicitly to zero later if the animation is already based on zero shouldn't impact the existing animation.

Reply:
Interesting. Can't say I fully understand it, but it does look like they're in sync and doesn't reset the existing animations, so cool.